docs/source/reference/yaml-spec.rst

```diff
@@ -403,14 +478,36 @@ If unspecified, defaults to ``false`` (on-demand instances).
 ``resources.disk_size``
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Disk size in GB to allocate for OS (mounted at ``/``).
+Integer disk size in GB to allocate for OS (mounted at ``/``) OR specify units.
 
 Increase this if you have a large working directory or tasks that write out large outputs.
 
+Units supported (case-insensitive):
+
+- B (bytes, 2^0 bytes)
+- K/KB (kilobytes, 2^10 bytes)
+- M/MB (megabytes, 2^20 bytes)
+- G/GB (gigabytes, 2^30 bytes)
+- T/TB (terabytes, 2^40 bytes)
+- P/PB (petabytes, 2^50 bytes)
+
+.. maybe instead of warning, have it be a note?
+.. warning::
+
+   The disk size will be rounded down to the nearest gigabyte. For example, ``1500MB`` will be rounded to ``1GB``.
```

Comment:
Just confirming: it's always rounded down, right? That is, 2000MB will still be rounded to 1G

Reply:
yes